README.md

```diff
@@ -287,6 +287,7 @@ var server = ParseServer({
     // 2. a callback function to be invoked to validate the password  
     validatorCallback: (password) => { return validatePassword(password) }, 
     doNotAllowUsername: true, // optional setting to disallow username in passwords
+    daysBeforeExpiry: 90, // threshold (days) for password expiry. Login fails if user does not reset the password within this period after signup/last reset. 
```

Comment:
Yes, I have seen that. I think the name specified there may not be really a best-practice but how it's label is displayed in the configuration screen (in windows group policies). 

I have chosen this name because you immediately know that the number is in "days". I have no particular preference though. Let me know if you really think it should be changed.